modules/040-control-plane-manager/monitoring/prometheus-rules/control-plane-manager.yaml

```diff
@@ -47,6 +47,23 @@
       description: |-
         Current kubernetes version "{{ $labels.k8s_version }}" is deprecated, and its support will be removed within 6 months
 
```

Comment:
`Description` here is the subsection of `labels`, but in the second alert - of `annotations`. So is `description` label or annotation?

Reply:
it's not. It was just collapsed by github